frontend/src/lib/components/appLogo/AppLogo.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+  import {page} from '$app/stores';
+  import type {AppLogoProps} from './AppLogo.type';
+
+  type $$Props = AppLogoProps;
+  export let alt: $$Props['alt'] = $page.data.appLabels.viewTexts.publisherName;
+  export let inverse: $$Props['inverse'] = false;
+  export let size: $$Props['size'] = 'md';
+
+  // Create class names
+  let classes: string;
+
+  // Predefined sizes
+  switch (size) {
+    case 'sm':
+      classes = 'h-20 pt-2';
+      break;
+    case 'lg':
+      classes = 'h-32 pt-6';
+      break;
+    default:
+      classes = 'h-24 pt-4';
+  }
+
+  // Merge classes into restProps
+  $$restProps.class = `${classes} ${$$restProps.class ?? ''}`;
+
+  // TODO: Use logo files defined in Strapi.
+  // TODO: Possibly allow passing props directly to the img elements
+</script>
+
+<!--
+@component
+Used to show the application's logo. The logo colour changes dynamically based
+on whether the light or dark mode is active.
+
+### Properties
+
+- See `AppLogo.type.ts` or code completion info.
+- Any valid attributes of the `<div>` element wrapping the light and dark
+  `<img>` elements
+
+### Usage
+
+```tsx
+<AppLogo size="lg"/>
+<div class="bg-primary">
+  <AppLogo inverse={true}/>
+</div>
+```
+-->
+
+<div {...$$restProps}>
+  <img
```

Comment:
why are you using one  `img` on top of the other? Just curious :).

Reply:
That's because I'm using the tailwind classes to decide whether to show the light or dark version of the logo based on dark mode. Only one of them is displayed at a time.

Reply:
but both of them get rendered. wouldn't make more sense to add a `{#if}` instead
<img width="520" alt="image" src="https://github.com/OpenVAA/voting-advice-application/assets/10437475/0c6fa50f-3efa-4257-a877-e42934e17276">

Reply:
That's surprisingly tricky, bc we need to create a media query listener for that, but it can be done. I'd suggest we use this quick version for now and create a separate issue for that, bc it'd make most sense to implement the dark mode listener as a global store, which other components can also import.

Would that be okay?